package.json

```diff
@@ -20,6 +20,7 @@
   },
   "devDependencies": {
     "electron": "^1.7.6",
-    "electron-packager": "^9.0.1"
+    "electron-packager": "^12.0.2",
```

Comment:
versions before 12.0.2 seem to have this issue: https://github.com/electron/electron-packager/issues/863